boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -131,6 +133,12 @@ oscilloscope.
 Channel 2 signal routed to resistance R881.
 Channel 3 signal routed to resistance R882.
 
+SPI
+---
+
+SPI1 on J35 is enabled for M7.
+R1217/R1218/R1219/R1220 should be soldered with 0R resistances.
```

Comment:
Do you mean need to rework the board? If yes, suggest to change it to be like the following:
The EVK board need to be reworked to solder R1217/R1218/R1219/R1220 with 0R resistances.

And is there any testcase available can be used to verify SPI on the board? If no device avaiable, whether can use loopback test: tests/drivers/spi/spi_loopback ?

Reply:
Thanks. Updated.
No device on board, I verified with both analyzer and loopback testcase.